Tasks/AzureRmWebAppDeploymentV4/deploymentProvider/BuiltInLinuxWebAppDeploymentProvider.ts

```diff
@@ -23,13 +23,18 @@ export class BuiltInLinuxWebAppDeploymentProvider extends AzureRmWebAppDeploymen
         tl.debug('Performing Linux built-in package deployment');
 
         if(this.taskParams.isFunctionApp) {
-            var linuxFunctionRuntimeSetting = linuxFunctionRuntimeSettingName + linuxFunctionRuntimeSettingValue.get(this.taskParams.RuntimeStack);
+            var linuxFunctionRuntimeSetting = "";
+            if(this.taskParams.RuntimeStack != null && this.taskParams.RuntimeStack != ""){
```

Comment:
it can be undefined too.
`if(this.taskParams.RuntimeStack)` should be good here